Creando el block que nos permita donar

Morban15 opened the pull request:
### Issues

_¿Qué se está resolviendo?_

Tenemos que hacer un block que contenga un botón que
 permita a los usuarios donar a la organización y debe tener
 un icono (cuadrado) y un texto (tiene un limite de expansión) de donar.
 Este block se debe ajustar a cualquier tipo de pantalla.
 También debe tener contenido en inglés y español.

#31 

### Todos

_¿Qué queda por hacer?_

- Codificar el archivo.html y .css
- Revisar
- Terminar
- MERGED!!!


### Áreas impactadas

_¿Qué páginas o componentes están afectados?_

- El archivo donate_block.html y .css
- Las páginas donde vamos a incluir el block donate
- Los archivo .yml

### Pasos para reproducir

_¿Qué necesitamos hacer para probar?_

- Usar el inspector para ajustarlo a diferentes tipos de pantallas

Morban15 committed:
Creando la carpeta "donate_block" y el archivo de HTML

Hemos creado la carpeta "donate_block" y el archivo "donate_block.html"
(aun no hemos codificado en este archivo).

Morban15 committed:
Creando el archivo css para la página donate

El archivo tiene por nombre donate_block.css y lo guardamos en la carpeta
donate_block.

Morban15 committed:
Añadiendo los archivos yml en En y Es

Hemos añadidos los archivos yml en ingles y en español llamados
¨donations.yml¨ y ¨donaciones.yml¨

Morban15 committed:
Agregando el href a los archivos de yml en ingles y español

Hemos agregado el href para el boton de donar.

Morban15 committed:
Codificando el archivo .html e incluyendo img

Hemos codificado en el archivo donate_block.html

Morban15 committed:
Añadiendo el donate_block.html a la página News

Hemos añadido el archivo donate_block.html al html de
la página news

Morban15 committed:
Añadiendo el archivo .css a la página site.css

Hemos añadido el archivo que lleva por nombre ¨donate_block.css¨
a la página ¨site.css¨.

Morban15 committed:
WIP

Haciendo algunos cambios.

Morban15 committed:
Trabajo terminado en donate_block

Hemos hecho algunos cambios para poder terminar.

misscs commented on app/modules/donate_block/donate_block.css:
```diff
@@ -0,0 +1,28 @@
+.donate-block {
+  display: flex;
+  border: 3px solid pink;
```
Podemos remover este estilo?

Morban15 replied:
Sí

misscs commented on app/modules/donate_block/donate_block.css:
```diff
@@ -0,0 +1,28 @@
+  .donate-block {
+    flex-direction: row;
+  }
+  .button-right {
```
Y si cambios el botón para la izquierda?

Morban15 replied:
Debo cambiar el nombre de esa clase 😹

misscs commented on app/modules/donate_block/donate_block.css:
```diff
@@ -0,0 +1,28 @@
+  border: 3px solid pink;
+  padding: 20px;
+  align-items: center;
+  flex-direction: column;
```
Podemos agrupar los estilos de flex arriba? Se hace más fácil leer y saber que esta pasando en el componente

Morban15 replied:
Sí

misscs commented on app/modules/donate_block/donate_block.css:
```diff
@@ -0,0 +1,28 @@
+
+.donate-content {
+  max-width: 20em;
+  flex-direction: column;
```
Puedes explicar porque necesitamos L11?

Morban15 replied:
Perdón, esto no es necesario 😂

misscs commented on app/modules/donate_block/donate_block.html:
```diff
@@ -0,0 +1,12 @@
+  <div class="button-right">
+    <a href="{{ donations.href_button }}">{{ donations.title }}</a>
+  </div>
+</div>
```
Ahora que sabes cómo crear macros :) Puedes crear una para `donate-block`?

Morban15 replied:
Está bien

misscs reviewed (changes requested):
@Morban15 Algunos cambios

Morban15 committed:
Quitando cosas no necesarias y ordenando codigos

He quitado el estilo del border porque no era necesario.
Tambien hemos ordenado los codigos para una mejor lectura.

Morban15 committed:
Ordenando codigos

Hemos ordenado los codigos para mejor lectura.

Morban15 committed:
Cambiando el nombre de la clase y quitando cosas no necesarias

He cambiado el nombre de la clase para la division del boton y
he quitado algo no necesario en css.

misscs commented:
@Morban15 Este PR está listo para mi revisión?

Morban15 commented:
@misscs
No.
En caso de tener un objeto y no un array, ¿Qué puedo utilizar? 😕 
para el loop.

Morban15 committed:
WIP

WIP

Morban15 committed:
Macro de donate_block terminado

He hecho algunos cambios para terminar el macro de donate_block.

Morban15 commented:
Terminado 😄

Morban15 committed:
Merge branch 'master' into donations

Morban15 committed:
Añadir el icono del modulo icon

He cambiado el icon-donate por el icon que está en modulo.

Morban15 committed:
Cambiando el nombre de la clase del botón para donar
